cmd/minikube/cmd/start.go

```diff
@@ -374,23 +375,33 @@ func provisionWithDriver(cmd *cobra.Command, ds registry.DriverState, existing *
 }
 
 func virtualBoxMacOS13PlusWarning(driverName string) {
-	if driverName != "virtualbox" || !detect.MacOS13Plus() {
+	if !driver.IsVirtualBox(driverName) || !detect.MacOS13Plus() {
 		return
 	}
-	driver := "hyperkit"
+	suggestedDriver := driver.HyperKit
 	if runtime.GOARCH == "arm64" {
-		driver = "qemu"
+		suggestedDriver = driver.QEMU
 	}
 	out.WarningT(`Due to changes in macOS 13+ minikube doesn't currently support VirtualBox. You can use alternative drivers such as docker or {{.driver}}.
     https://minikube.sigs.k8s.io/docs/drivers/docker/
     https://minikube.sigs.k8s.io/docs/drivers/{{.driver}}/
 
     For more details on the issue see: https://github.com/kubernetes/minikube/issues/15274
-`, out.V{"driver": driver})
+`, out.V{"driver": suggestedDriver})
+}
+
+func vmwareUnsupported(driverName string) {
+	if !driver.IsVMware(driverName) {
+		return
+	}
+	exit.Message(reason.DrvUnsupported, `Due to security improvements to minikube the VMware driver is currently not supported. Available workarounds are to use a different driver or downgrade minikube to v1.29.0.
```

Comment:
we would accept community contributions to fix this issue,

Reply:
Done